src/coord/src/coord.rs

```diff
@@ -312,6 +332,7 @@ where
     }
 
     pub fn get_ts(&mut self) -> Timestamp {
+        self.need_advance = true;
```

Comment:
I'd love to have comments here, esp on a public method. What does this do, who should call it, what should they expect, etc.

Reply:
Good call—added a comment and made this non-public.